app/lib/navigation.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:app/screens/history/history.dart';
+import 'package:app/screens/in_progress/in_progress.dart';
+import 'package:flutter/material.dart';
+
+class NavigationBarController extends StatefulWidget {
+  @override
+  _NavigationBarControllerState createState() =>
+      _NavigationBarControllerState();
+}
+
+class _NavigationBarControllerState extends State<NavigationBarController> {
+  int _selectedIndex = 0;
+  final List<Widget> _widgetOptions = <Widget>[
+    InProgress(),
+    TravelHistory(),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Center(
+        child: _widgetOptions.elementAt(_selectedIndex),
+      ),
+      bottomNavigationBar: BottomNavigationBar(
+        items: [
+          BottomNavigationBarItem(
+            label: 'In Progress',
+            icon: Icon(Icons.star_border_outlined),
+          ),
+          BottomNavigationBarItem(
+            label: 'Travel History',
+            icon: Icon(Icons.history_edu_outlined),
+          ),
+        ],
+        currentIndex: _selectedIndex,
+        //Update the state when user selected an item
```

Comment:
Do you have pre-commit hooks set up? Looks like missing whitespace in the comment

- [ ] Set up pre-commit hooks

Reply:
Yea, pre-commit is installed

Reply:
Fixed the issue, I forgot to run this command `pre-commit install`

Reply:
I didn't see any change here. I think because you already committed, it didn't run again? 

- [ ] Try `pre-commit run --all-files` and make sure it doesn't make changes.
- [x] I'm going to look into adding the pre-commit checks (the not-flutter ones) as another action (sorry for making you the guinea pig)